spec/v1/statement.md

```diff
@@ -67,6 +67,15 @@ Additional [parsing rules] apply.
 > set-but-empty. MAY be omitted if `predicateType` fully describes the
 > predicate.
 
+## Change History
+
+### v1.1
+Clarified that subjects are assumed to be immutable.
```

Comment:
```suggestion

Clarified that subjects are assumed to be immutable.
```

Reply:
I'm a bit confused by this suggestion.  In the most recent commit there is a blank line after `### v1.1` and before this sentence.

I don't think you're suggesting it needs to blanks.  So perhaps this was on an old commit?  Maybe the suggestion is something else?

Reply:
There's no blank line in this file (statement.md) while there is one in digest_set.md.

Reply:
Oh, weird, in my screen that wasn't present. Nevermind.

Reply:
Looks like GitHub was serving you an old commit.   An extra line was added in https://github.com/in-toto/attestation/pull/338/commits/efef96f0b83b682f34b91bb7ab1e08fa86a82cc9